dashboard/Home.py

```diff
@@ -1,18 +1,248 @@
 '''A script that creates the home page for the dashboard.'''
 
 from os import environ as ENV
+from datetime import datetime, timedelta
 
+import pandas as pd
+import altair as alt
 import streamlit as st
+from dotenv import load_dotenv
+
```

Comment:
Get rid of blank line; they're all third-party libraries, and so should be in the same block